tokens/figma/README.md

```diff
@@ -0,0 +1,61 @@
+# Project Overview
+
+This project is designed to update Figma variables based on a JSON input, primarily focused on managing brand themes, colors, and other design tokens. The project retrieves existing variables from Figma, processes the provided JSON data, and updates or creates new variables in collections "Mode" and "Brand".
+
+## Features
+
+- **Fetch Existing Figma Data**: Retrieves the existing variables and collections from Figma.
+- **Process JSON Data**: Extracts theme and token data from provided JSON files for each brand.
+- **Update or Create Variables**: Adds new variables or updates existing ones based on the brand's light and dark themes.
+- **Handle Variable Modes**: Ensures each brand's mode (e.g., "Light", "Dark") is updated or created in the Figma "Brand" collection.
+- **Support for Multiple Brands**: Processes multiple brands, mapping each brand's unique variables into Figma's collections.
+
+## Setup
+
+### Environment Variables:
```

Comment:
Aren't you missing the brand variables here? (like `MOVISTAR_FILE_KEY`)

Reply:
Added